roles/openshift_logging_elasticsearch/templates/passwd.j2

```diff
@@ -0,0 +1,2 @@
+"{{_user_name}}":
```

Comment:
`_user_name` and `_user_passwd` may be too generic of variable names (possibly affected by creep).
Could you make this a little more specific? Maybe just prefix it with `logging_passwd_file`